webapp/src/components/SigninForm/index.tsx

```diff
@@ -65,7 +65,7 @@ function SigninForm() {
         </Button>
         <Grid container>
           <Grid item xs>
-            <Link to="/mot-de-passe-oublie" component={NavLink}  variant="body2">
+            <Link to="/forgot-password" component={NavLink}  variant="body2">
```

Comment:
in `appPaths.ts` file

```ts
export const FORGOT_PASSWORD_PATH = "/forgot-password"
```

```suggestion
            <Link to={FORGOT_PASSWORD_PATH} component={NavLink}  variant="body2">
```